src/server.js

```diff
@@ -44,24 +46,33 @@ const buildHtmlString = (body, cache) => {
 // Shared serverModel
 // You can also hardcode / stub parts of the model here
 const serverModel = new falcor.Model({
-  cache: {
-    articles:articles,
-    authors:authors,
-  },
+  cache: data,
   source: new FalcorRouter(),
 }).batch();
 
 // Asynchronously render this application
 // Returns a promise
 const renderApp = (renderProps) => {
-  const falcorPaths = _.compact(renderProps.routes.map((route) => {
+  let falcorPaths = _.compact(renderProps.routes.map((route) => {
     const component = route.component;
     if (component.prototype instanceof FalcorController) {
       return component.getFalcorPath(renderProps.params);
     }
     return null;
   }));
 
+// Merging pathsets
+var temp = [];
```

Comment:
Is the indenting here a little messed up? 

Also I think this entire block can be replaced with `_.flatten()`

https://lodash.com/docs#flatten